script.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable max-classes-per-file */
 /* eslint-disable no-console */
+/* eslint-disable no-unused-vars */
```

Comment:
- [OPTIONAL] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylaistic and programming errors against the rules it knows. 💯 😉